cmd/commands.go

```diff
@@ -0,0 +1,193 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/StackExchange/dnscontrol/models"
+	"github.com/urfave/cli"
+)
+
+// categories of commands
+const (
+	catMain  = "main"
+	catDebug = "debug"
+	catUtils = "utility"
+)
+
+// Run will execute the CLI
+func Run(version string) error {
+	app := cli.NewApp()
+	app.Version = version
+	app.Name = "dnscontrol"
+	app.HideVersion = true
+	app.Usage = "dnscontrol is a compiler and dsl for managing cloud dns zones"
```

Comment:
"dnscontrol is a compiler and DSL for managing dns zones" (cap DSL and remove "cloud")